packages/components/src/ComboBox/src/ComboBox.tsx

```diff
@@ -90,7 +89,7 @@ export interface ComboBoxProps<T extends object> extends StyledComponentProps<Om
      * The size of the select.
      * @default "sm"
      */
-    size?: ResponsiveProp<ComboBoxSize>;
+    size?: ResponsiveProp<InputSize>;
```

Comment:
Just wondering if that's the naming we want? InputSize in an ComboBox component. Maybe FieldSize?

Reply:
Or you could do similar to EmbeddedButton? Feel free to poke me on Slack. Maybe not needed neither.

```
export type ComboBoxSize = InputSize;
```